src/Serialization/Deserializable.php

```diff
@@ -0,0 +1,8 @@
+<?php declare(strict_types=1);
```

Comment:
It looks like the DeserializableEvent interface does not extend this interface. Is there a reason for this?